### PR TITLE
INFRSTR-368 Docker modernization

### PR DESCRIPTION
**Pull request by jonespm**
Removed old config that waasn't likely used
Put the Dockerfile at the top level with an updated readme Tested that this mimially works with the current version of Sakai

TODO ideas in a future fix:

* Use docker-compose.yml on the top level to make starting this up easier This would also allow for a persistent volume for MariaDB

* Split tomcat from the Dockerfile and have it run separately.

* Verify if we still need the docker-maven-chrome or if we can use the regular maven to build

* Create a .dockerignore (if needed)

* Build from this directory instead of checking out again. I skipped this for now because it would have to clean up before building

* Verify and perform additional cleanup on the container (if possible)

### CODE REVIEW

**Comment by jonespm** [comment]
@ern @csev 

I validated that this runs and starts up locally. It cleans up a lot in the docker directory that was outdated while providing a file and working route to build and run from source in Sakai 17. 

There's additional work here if people use this but this feels like a minimum viable product.

**Comment by csev** [comment]
I checked it out - followed the directions and it started and ran.  I suggest merge.

**Review by csev — APPROVED** [review]
Works when I check it out and run it.  I say put it in and folks can improve later.  So much better than what is there.

File: docker/README.md [hunk 1]
```diff
@@ -1,38 +1,54 @@
...
+```
+	# May want to include an opt for docker rm sakai-mariadb
+	# Start it if we've already created it, unless we want to re-create
+	docker run -p 127.0.0.1:53306:3306 -d --name="sakai-mariadb" --pull always \
+	    -e "MARIADB_ROOT_PASSWORD=sakairoot" \
+	    -v "./mysql/scripts:/docker-entrypoint-initdb.d" \
+	    -d mariadb:10 --lower-case-table-names=1 || docker start "sakai-mariadb"
+```
+## Starting up Sakai and linking to Maria
+    `docker run --rm -p 8080:8080 --name sakai-tomcat --link sakai-mariadb sakai`
```
Comment (csev) [inline docker/README.md:54]: You might want to add instructions as to how to connect ot the database.  I did this:

    mysql -u root -P 53306 -p -h 127.0.0.1

Password `sakairoot`

Comment (jonespm) [reply docker/README.md:54]: This was just in this SQL that I have in here as a default to run and prep the DB. Probably just some legacy I'd found in documentation somewhere.

https://github.com/sakaiproject/sakai/pull/13113/files#diff-ec7a841b95a0c9984fc69c019a3b384e7a7fade863b32ddc4e3c76b92506f278

I can clean up these unused databases from the script.

Comment (csev) [reply docker/README.md:54]: The Maria bit was very nice to have.  Easy for you - harder for us docker beginners - thanks.

Comment (jonespm) [reply docker/README.md:54]: I updated the documentation to add running mysql (Maria) directly from the container and also check that it's running and avoid creating the container if it already exists. 

I also updated the script to remove the old databases.

**Review by ottenhoff — APPROVED** [review]
Nice

**Pull request merged**